### PR TITLE
Docs/Missing Feature Documentation

### PR DESCRIPTION
**Pull request by sdreyer**
# Description

Adds additional information around:
1. Timestamps
2. Scheduling
3. Kubernetes configuration
4. Status Checking
5. Waits
6. Updates to serving format
7. Chaining Transformations

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have fixed any merge conflicts

### CODE REVIEW

File: docs/getting-started/serving-for-inference-and-training.md [hunk 1]
```diff
@@ -46,7 +55,7 @@ import featureform as ff
 
 client = ff.ServingClient(host)
 buffer_size = 1000
-for features, labels in client.training_set(name, variant).shuffle(buffer_size):
+for row in client.training_set(name, variant).shuffle(buffer_size):
```
Comment (simba-git) [inline docs/getting-started/serving-for-inference-and-training.md:58]: does the thing above not work? I think it would assuming its not batched

**Review by simba-git — COMMENTED** [review]
Great, just one question

File: docs/getting-started/serving-for-inference-and-training.md (hunk 1 above)
Comment (sdreyer) [reply docs/getting-started/serving-for-inference-and-training.md:58]: lemme check

Comment (sdreyer) [reply docs/getting-started/serving-for-inference-and-training.md:58]: `Traceback (most recent call last):
  File "../../../featureform/train.py", line 7, in <module>
    for feature, label in dataset:
TypeError: cannot unpack non-iterable Row object`

**Pull request merged**